src/utils/hendelseFilteringUtils.tsx

```diff
@@ -92,17 +121,8 @@ export const filterOnPersonregister = (
   }
 
   const filtered = Object.entries(personregister).filter(([, personData]) => {
-    return (
-      (!filter.onskerMote || personData.harMotebehovUbehandlet) &&
-      (!filter.arbeidsgiverOnskerMote ||
-        personData.harOppfolgingsplanLPSBistandUbehandlet) &&
-      (!filter.dialogmotekandidat || personData.dialogmotekandidat) &&
-      (!filter.ufordeltBruker || !personData.tildeltVeilederIdent) &&
-      (!filter.dialogmotesvar || personData.harDialogmotesvar) &&
-      (!filter.aktivitetskrav || personData.aktivitetskravActive) &&
-      (!filter.behandlerdialog || personData.harBehandlerdialogUbehandlet) &&
-      (!filter.aktivitetskravVurderStans ||
-        personData.harAktivitetskravVurderStansUbehandlet)
+    return Object.keys(filter).every((key) =>
+      matchesFilter(key as HendelseTypeFilterKey, filter, personData)
```

Comment:
Blir det forsatt et "snitt"-filter her? Altså hvis de huker av to filtere, så viser den kun personer som har begge oppfylt?

Reply:
Mener det fortsatt skal bli riktig ja. Er en test på det i hendelseFilteringUtilsTest også.

Reply:
Bra 🙌🏼 Synes generelt det er litt vanskelig logikk rundt disse filterene, og er ikke like lett å lese alle `Object.` og `!filters` etc. 
Men `every` burde gjøre susen!